includes/multi-currency/Notes/NoteMultiCurrencyAvailable.php

```diff
@@ -40,7 +41,7 @@ class NoteMultiCurrencyAvailable {
 	 * Get the note.
 	 */
 	public static function get_note() {
-		$note_class = \WC_Payment_Woo_Compat_Utils::get_note_class();
+		$note_class = Note::class;
```

Comment:
'$note = new Note();' - this an next assignments can be combined into one (also in other notes classes we are touching by this PR)

Reply:
Correspondingly. '$note_class::E_WC_ADMIN_NOTE_INFORMATIONAL' below becomes 'Note::E_WC_ADMIN_NOTE_INFORMATIONAL'

Reply:
Thanks for the guidance! It did not strike me earlier. I have made the changes! I hope it is okay now. 

9434390